index.js

```diff
@@ -169,6 +185,34 @@ express()
     }
   })
 
+  .get('/editProfile', jsonParser, async (req, res) => {
+    try {
+      //var userID = userProfile ? userProfile.id : 1;
+      //const client = await pool.connect();
+      //const result = await client.query(`SELECT name, email, postal_code, email_freq FROM Users where user_ID = '${userID}'`);
+      const client = await pool.connect();
+      const result = await client.query(`SELECT name, email, postal_code, email_freq FROM Users where user_ID IN ('1')`);
```

Comment:
See above comment about updated to google userID

Reply:
Will do thank you!